lact-schema/src/tests.rs

```diff
@@ -34,12 +35,15 @@ fn controllers_response() {
 
 #[test]
 fn error_response() {
-    let expected_response = json!({
-        "status": "error",
-        "data": "my super error"
-    });
+    let response = Response::<()>::from(anyhow!("my super error"));
 
-    let response = Response::<()>::Error("my super error".to_owned());
+    let json_value = serde_json::to_value(&response).unwrap();
+    let parsed_response: Response<()> = serde_json::from_value(json_value).unwrap();
 
-    assert_eq!(serde_json::to_value(response).unwrap(), expected_response);
+    match (response, parsed_response) {
+        (Response::Error(first), Response::Error(second)) => {
+            assert_eq!(second.to_string(), first.to_string())
+        }
+        _ => unreachable!(),
+    };
```

Comment:
The goal of these tests is to check the actual json representation of responses, not just that serialization is working properly. So it should still have an `expected_response` json literal that's compared against the serialized form. 
I think here it would be especially useful to check the json representation of an error with a backtrace. If i'm understanding it right, it should be a nested error object.

Reply:
@ilya-zlobintsev actual JSON representation is implementation defined in the serde-error crate. I think it doesn't matter what actual format is uses. You would need to update it after every serde-error dependency update.

Reply:
What matters is that it produces same output after serialization/de-serialization.

Reply:
I'm not sure how to implement that test properly without hard pinning the dependency

Reply:
@ilya-zlobintsev do you have any suggestions?

Reply:
While the main purpose of the JSON API is communication between the GUI and the daemon, it's not an arbitrary internal API as it's possible to make third party scripts that interact with it. For this reason, I would like to avoid changing the JSON representation of requests/responses when possible. Of course this PR changes the error format, which is fine, but it would be best to avoid accidentally changing it again in the future.

In practice I don't think it's going to be an issue, serde-error didn't have any functional changes in [4 years](https://github.com/Ekleog/serde-error/commits/master/), but I would still like to have a test that defines what the error representation is.

Reply:
@ilya-zlobintsev Okay, I pinned the dependency and updated test. I understand your point